ENV_VARS.md

```diff
@@ -92,6 +92,10 @@ applications.
   on which the Express daemon of the Geoprocessing service will listen.
   If running the API on the same host as the Geoprocessing application, you 
   need to modify at least one.
+* `CLEANUP_CRON_INTERVAL` (string, optional, default is 0 0-23/6 * * *):
```

Comment:
if this works like Vixie cron, I think it could even be as below (but leave it as is, just neckbearding here)

```suggestion
* `CLEANUP_CRON_INTERVAL` (string, optional, default is `0 */6 * * *`):
```